### ISSUE TITLE
Remember "columns" view (i.e Company) 

### ISSUE THREAD

**Issue by jovibroz** [grokability/snipe-it#8888]
**Server (please complete the following information):**
 - Snipe-IT Version 5.0.11
 - OS: [Ubuntu 20.04]
 - Web Server: [Apache]
 - PHP Version 7.4.3

Would be nice if the columns view was remembered by your web browser of choice.
I'm on Safari mostly. 

High Sierra 10.13.6
Safari 13.1.2

**Comment by snipe** [comment]
They are remembered - we set a cookie.

**Comment by jovibroz** [comment]
Haha wicked. Works now in Safari. Right after I tried in Chrome (where it worked). Then back to Safari and "company" column sticks in there as well. Could that be? First logging in with Chrome needed for Safari to work? Computeeeeers woha! Whatever :) I like it. Case closed I guess.

**Comment by UserGoodtm** [comment]
In fact, it pisses me off when columns are constantly being reset and you have to re-mark them every time so that the information becomes available for viewing

**Comment by jovibroz** [comment]
yeah, it's definitely not remembered after logging out but maybe that's how it goes.

**Comment by Midnex** [comment]
I would like if these are not remember via cookie, but internal database table. I often use 3 different machines when working on recording assets, and I am having to set them up on every single machine. Kind of a pity. And of course every user has to set theirs up as well. I can't set defaults.

**Comment by snipe** [comment]
We'd definitely take a PR for this, but it's pretty low on our roadmap in terms of upcoming features.

**Comment by wewhite** [comment]
Shouldn't the default columns displayed be exactly what the custom field sets are for that model, and nothing else?
If there are not field set defined, then show all columns.

I have custom field set for vehicles, and would like to see only those fields when viewing the asset. 
Do not need to see CPU, RAM, etc. other custom fields that have nothing to do with a vehicle.
